README.md

```diff
@@ -38,3 +38,22 @@ Don't do link check:
 ```Bash
 $ npx respec-validate --no-links spec.html
 ```
+
+## FAQ
+
+### Can I make it run faster?
+
+If you already have Chromium installed, you can set the following environment variables to avoid downloading Chromium unnecessarily.
```

Comment:
```suggestion
If you already have Chrome installed, you can set the following environment variables to avoid downloading Chromium unnecessarily.
```